src/test/scala/scynamo/ScynamoTest.scala

```diff
@@ -1,15 +1,23 @@
 package scynamo
 
-import software.amazon.awssdk.services.dynamodb.model.{AttributeValue, GetItemResponse}
+import cats.data.{EitherNec}
+
+import software.amazon.awssdk.services.dynamodb.model.{AttributeValue, GetItemResponse, QueryResponse}
 
 class ScynamoTest extends UnitTest {
   "Scynamo" should {
-    "return None if the response has no item" in {
+    "return None if the get item response has no item" in {
       val response = GetItemResponse.builder().build()
 
       Scynamo.decodeGetItemResponse[Map[String, AttributeValue]](response)
     }
 
+    "return an empty Sequence if the query response has no items" in {
+      val response = QueryResponse.builder().build()
+
+      Scynamo.decodeQueryResponse[Map[String, AttributeValue]](response)
```

Comment:
I'm not familiar with the code but… shouldn't there be a `should` somewhere? To… _test_ something? 😄

Reply:
😆  I can't stop laughing about this 😆   Added _the missed since ever_ checks in 96430ac